tools/pipelines/build-client.yml

```diff
@@ -13,6 +13,8 @@ parameters:
   values:
     - none
     - prerelease
+    - alpha
+    - beta
```

Comment:
I think I maybe left feedback on the wrong PR - sorry! Anyway, I think this should be a separate parameter from the release param. maybe something like:

```yaml
parameters:
- name: packageTypesOverride
  displayName: Change package types (default = no change)
  type: string
  default: none
  values:
    - none
    - alpha
    - beta
    - public
    - untrimmed
```

(maybe the default will change to `public` at some point?)